portfolio/src/main/webapp/css/style.css

```diff
@@ -132,4 +132,54 @@ html, body {
     background: none;
     border: none;
     padding: 0;
-}
\ No newline at end of file
+    cursor: pointer;
+}
+
+/* Like button adapted from https://codepen.io/kieranfivestars/pen/PwzjgN */
+.smile-container {
+    float:right;
+    padding-right: 10px;
+}
+
+.smile {
+    cursor:pointer;
+    color:#aaa;
+    transition:.2s;
+    font-size: 25px !important;
+}
+
+.smile:hover {
+  color:#666;
+}
+
+.smile.press {
+  animation: size .4s;
+  color: pink;
+}
+
+@keyframes fade {
+  0% {color:#transparent;}
+  50% {color:#e23b3b;}
+  100% {color:#transparent;}
+}
```

Comment:
I'm just curious, what does the @ in keyframes mean?

Reply:
It specifies an "at-rule" in CSS (such as @media, @import, etc). In this case, @keyframes lets us mark what happens at each point of an animation.